client/my-sites/promote-post-i2/utils/index.ts

```diff
@@ -391,3 +400,53 @@ export const cvsStatsDownload = ( csvData: string, fileName: string = 'report.cs
 	link.click();
 	URL.revokeObjectURL( link.href );
 };
+
+export const getPaymentStatusBadgeColor = ( status?: string ) => {
+	switch ( status ) {
+		case paymentStatus.COMPLETED: {
+			return 'info-green';
+		}
+		case paymentStatus.FAILED: {
+			return 'error';
+		}
+		case paymentStatus.PENDING: {
+			return 'error';
```

Comment:
Wondering if pending should be info blue. I know in WooCommerce it's grey rather than red like failed.

Reply:
i swithed it to grey.. i think it makes more sense than blue..

i think this is a trivial issue. the average user wont have any pending (it will try a few times to process the payment and it will become either success or failed)
<img width="368" height="226" alt="Screenshot 2025-08-05 at 12 43 25" src="https://github.com/user-attachments/assets/2bc3a562-60f8-4ed1-b3eb-fc275178f1d5" />

but let me know what you think @j6ll